test-apps/display-performance-test-app/src/frontend/TestRunner.ts

```diff
@@ -1146,10 +1154,11 @@ function getOtherProps(vp: ScreenViewport): string {
 
 function getViewFlagsString(test: TestCase): string {
   let vfString = "";
-  for (const [key, value] of Object.entries(test.viewport.viewFlags)) {
+  const renderMode = test.viewport.viewFlags.renderMode;
```

Comment:
Several of these (lighting, visible edges, etc) are always true or always false for certain render modes. I omit them if they cannot vary for the given render mode.